src/cli/doctor-download.js

```diff
@@ -8,6 +8,9 @@ commander
   .option("-f, --file [file]", "location of file to save objects")
   .option("-d, --dir [dir]", "location of directory to save objects")
   .option("-n, --name [name]", "name of specific object to be download")
+  .option("-l, --level [level]", "level of the vdr")
```

Comment:
Hey @ApoorvaTiwari, Jamie asked me to review & test this PR.  For testing, it would be helpful to document the added download functionality in the readme.  I know we previously chatted about adding it after the upload functionality was completed, but I want to make sure I am testing the new functionality in the manner you expect.

Reply:
Sure @nick-bair - I am adding the readme so it will be easy for you to test it out. Thanks for pointing it out.